cluster/photon-controller/util.sh

```diff
@@ -781,31 +789,87 @@ function verify-ssh-prereqs {
 function verify-photon-config {
     kube::log::status "Validating Photon configuration..."
 
-    fail=0
-    rc=0
+    # We don't want silent failure: we check for failure
+    set +o pipefail
+
+    verify-photon-flavors
+    verify-photon-image
+    verify-photon-tenant
+
```

Comment:
Create VM call does all verification for you. I would not recommend making these calls separately. If you find something missing/need better messages in that call you can fix it there.
It will be a kind of repetition for that validation here.